systemd/Dockerfile.wheezy.tpl

```diff
@@ -0,0 +1,32 @@
+FROM #{FROM}
+
+# Install Systemd
+
+RUN echo "deb http://ftp.debian.org/debian/ wheezy-backports main contrib non-free" >> /etc/apt/sources.list
+RUN apt-get update && apt-get -t wheezy-backports install -y systemd \
```

Comment:
-t to retrieve packages from wheezy-backports repository. http://manpages.ubuntu.com/manpages/lucid/man8/apt-get.8.html